src/calculateRentalCost.js

```diff
@@ -29,6 +29,21 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const pricePerDay = 40;
+  const discountFrom3Days = 20;
+  const discountFrom7Days = 50;
+
+  let totalCosts = days * pricePerDay;
+
+  if (days >= 7) {
+    totalCosts -= discountFrom7Days;
+  }
+
+  if (days >= 3 && days < 7) {
```

Comment:
```suggestion
  if (days >= 3) {
```
ты уже проверил что число < 7 в предыдущем if. Если сразу из if будешь делать return - эта проверка не нужна

Reply:
ок дякую. Я виправив. Тільки там де < 7 return ? Всього 2 return правильно?